src/main/java/org/wildfly/security/util/PasswordBasedEncryptionUtil.java

```diff
@@ -153,6 +176,72 @@ private static boolean isBase64(Alphabet alphabet) {
         return Normalizer.normalize(new String(buffer), Normalizer.Form.NFKC).getBytes(StandardCharsets.UTF_8);
     }
 
+    private static byte[] picketBoxBase64Decode(String picketBoxBase64) {
```

Comment:
It has to duplicate logic from PicketBox. It has the iteration on bytes different (padded from beginning) than Elytron.